feature-pack-build-maven-plugin/src/main/java/org/wildfly/build/plugin/FeaturePackBuildMojo.java

```diff
@@ -130,7 +130,7 @@ private void copyContents() {
 
             final Path path = Paths.get(baseDir.getAbsolutePath());
 
-            final Path baseFile = Paths.get(configDir.getAbsolutePath() + "/src/main/resources");
+            final Path baseFile = project.getBasedir().toPath().resolve(configDir.getPath());
```

Comment:
This won't work. It's combining two directories which would more than likely result in an invalid directory.